mathics/builtin/colors/color_operations.py

```diff
@@ -201,26 +203,32 @@ def eval(self, input, colorspace, evaluation):
 
 class ColorNegate(Builtin):
     """
-    <url>
-    :WMA link:
-    https://reference.wolfram.com/language/ref/ColorNegate.html</url>
+    Color Inversion (<url>
+    :WMA:
+    https://reference.wolfram.com/language/ref/ColorNegate.html</url>)
 
     <dl>
-      <dt>'ColorNegate[$image$]'
-      <dd>returns the negative of $image$ in which colors have been negated.
-
       <dt>'ColorNegate[$color$]'
-      <dd>returns the negative of a color.
+      <dd>returns the RGB color when it is subtracted from white.
 
-      Yellow is RGBColor[1.0, 1.0, 0.0]
-      >> ColorNegate[Yellow]
-       = RGBColor[0., 0., 1.]
+      <dt>'ColorNegate[$image$]'
+      <dd>returns an image where each pixel in the image is replaced \
```

Comment:
Maybe `returns an image where each pixel has its color negated`?

Reply:
Yes, that is better.